examples/specific/membergroups.cfg

```diff
@@ -0,0 +1,11 @@
+PROJECT_NAME     = "Member Groups"
+OUTPUT_DIRECTORY = membergroups
+GENERATE_LATEX   = NO
+GENERATE_MAN     = NO
+GENERATE_RTF     = NO
+CASE_SENSE_NAMES = NO
+INPUT            = membergroups.h
+QUIET            = YES
+JAVADOC_AUTOBRIEF = YES
+GENERATE_HTML = NO
+GENERATE_XML = YES
```

Comment:
Missing newline at end of file. I'll fixup this before merging as it's super minor.